components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -3985,7 +3985,7 @@ private String getUserRoleListQuery() throws APIManagementException {
         }
         rolesQuery.append(")");
         if(log.isDebugEnabled()) {
```

Comment:
shall we format this code as well

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/12201/commits/1136a795360198388c6b4bd97029c20b0f898832

Thanks and Regards